src/content/en/fundamentals/engage-and-retain/push-notifications/good-notification.markdown

```diff
@@ -0,0 +1,243 @@
+notification that gets the most attention. For example, in western languages
+text is read from left to right and from top to bottom. So a messaging app would
+might put the name of the sender at the top and left.
+
+Good:
+<img src="images/pete-replied.png" width="316" height="73" />
+
+Bad:
+<img src="images/new-missive.png" width="316" height="76" />
+
+### Keep the message short {#keep-it-short}
+
+Notifications aren't emails. The intent of notifications is to tease users so
+they'll open your app. The `PushMessageData` object lets you send data to the
+user immediately, but   you may not want to show all of that data to the user,
+especially if additional   data could accumulate on the server after the
+notification is sent.
+
+## Relevant {#relevant}
+
+A relevant message is one about people or subjects the user cares about.
+
+### prefer logged-in users {#prefer-logged}
+
+Only ask for notification permissions from users who are logged in.
+If you don't know who your users are it's hard to send them relevant
+notifications. And if notifications aren't relevant, users might  call them
+spam.
+
+### Don't repeat information {#dont-repeat}
+
+You have little space to convey much information. Don't waste it by duplicating
+information between parts of the notification. The duplicate information may be
+relevant information, but removing duplication gives you extra room for other
+information. For example if your title contains the day of the week,
+don't also list it in the body.
+
+Good:
+<img src="images/notification-no-dup-content.png" width="316" height="73" />
+
+Bad:
+<img src="images/notification-dup-content.png" width="317" height="85" />
+
+Also, if the app is open, it's possible the new information is already on the
+screen. Instead of a notification, use the application UI to notify the user.
+
+### Don't advertise your native app {#dont-advertise-native}
+
+The point of service workers, the technology behind push notifications, is that
+you can avoid the time and expense of writing an application separate from your
+website. A user who has both your service worker and your native app may get
+duplicate notifications unless you write server-side code to prevent it. You can 
+avoid the problem completely; don't encourage users to run both.
+
+### Don't advertise {#dont-advertise}
+
+You'll have opportunities to monetize the user experience once they're in your
+app. Don't blow it by spamming your users when they're not. If you spam your
+users with notifications, you may lose them altogether.
+
+### Don't Include your Website Name or Domain {#no-website}
+
+Notifications already contain your domain name, and space is tight anyway.
+
+![Domain name in a Chrome notification.]()  
+![Domain name in a Firefox notification.]()
+
+### Make the icon contextual {#contextual-icon}
+
+Icons should convey something about the message they accompany. Consider this
+example.
+
+<img src="images/contextual-icon.png" width="324" height="117" />
```

Comment:
These images seem to be flipped. You say the icon doesn't tell us who it's from, but it's PK's face.